include/unifex/detail/unifex_fwd.hpp

```diff
@@ -37,20 +28,20 @@ namespace unifex {
 
   using detail::_ignore;
 
-  namespace _execute_cpo {
-    extern const struct _fn execute;
-  } // namespace _execute_cpo
-  using _execute_cpo::execute;
+  namespace _execute::_cpo {
+    struct _fn;
+  } // namespace _execute::_cpo
+  extern const _execute::_cpo::_fn execute;
 
   namespace _submit_cpo {
     extern const struct _fn submit;
   } // namespace _submit_cpo
   using _submit_cpo::submit;
 
-  namespace _connect_cpo {
-    extern const struct _fn connect;
-  } // namespace _connect_cpo
-  using _connect_cpo::connect;
+  namespace _connect::_cpo {
```

Comment:
Why make this change in this PR but not a similar one for submit?

Reply:
Because I was touching connect but not submit, no other reason. I've been making this change in dribs and drabs as I maintain code. I can do submit too if you like.